Lib/test/test_pdb.py

```diff
@@ -2565,6 +2565,9 @@ def test_pdb_issue_gh_101673():
     ...     '!a = 2',
     ...     'll',
     ...     'p a',
+    ...     'u',
```

Comment:
Does it make sense to check here that the local variable is not showing up in the other frame?

Reply:
Yeah. I was surprised that we actually never test accessing local variables after switching frames. But this is a meaningful test I guess to make sure the local variable is not leaked (say to globals?).